### ISSUE TITLE
Use Redoc and deploy as GitHub pages

### ISSUE THREAD

**Issue by gabor-boros** [opcotech/elemo#137]
### Summary

Using Redocly's free plan is not providing support for public API documentation. However, it has an open source API documentation tool, [Redoc](https://redocly.com/docs/redoc/) that could be used. Switch to Redoc and publish the API documentation as GitHub pages.

### Example Use-cases

- Allow checking a well-formatted API documentation by anyone.

### Drawbacks

N/A

**Comment by Sarah-2003** [comment]
Hi, I'm a new contributor and would love to work on this issue. Could you kindly assign it to me?

**Comment by gabor-boros** [comment]
@Sarah-2003 It may happen you need access to some stuff. If that's the case, feel free to ping me here or on discord 😊

**Comment by Sarah-2003** [comment]
@gabor-boros I have sent u a request over discord. Please accept my request.

**Comment by gabor-boros** [comment]
Hey @Sarah-2003 are your still working/planning to work on this?

**Comment by gabor-boros** [comment]
@Sarah-2003 happy to see you around! I just assigned this back to you ☺️

**Comment by Sarah-2003** [comment]
Can you please guide me around this.

**Comment by gabor-boros** [comment]
@Sarah-2003 yes, sure! I did update the description, is this feels more comfortable or would you like me to elaborate on it? 😇
